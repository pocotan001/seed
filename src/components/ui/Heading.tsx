import Text from "./Text";

const Heading = Text.withComponent("h1").extend`
  line-height: normal;
`;

Heading.defaultProps = {
  fz: "2rem",
  fw: 700,
  c: "grey800"
};

export default Heading;
