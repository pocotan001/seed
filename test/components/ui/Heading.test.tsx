import * as React from "react";
import * as renderer from "react-test-renderer";
import Heading from "~/components/ui/Heading";

describe("<Heading>", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Heading>alo</Heading>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
