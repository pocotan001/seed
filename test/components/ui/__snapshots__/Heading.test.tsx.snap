// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading> renders correctly 1`] = `
<h1
  className="sc-htpNat izMNip"
>
  alo
</h1>
`;
